### ISSUE TITLE
citeseer dataset loss is nan

### ISSUE THREAD

**Issue by CEfanmin** [danielegrattarola/keras-gat#14]
hi, thanks for your code. 
There is a problem that when i use citeseer dataset , the training loss is nan.

**Comment by danielegrattarola** [comment]
Hi, thanks for pointing this out. I'll try to look into it as soon as possible, it's probably related to the function used to load the data.

**Comment by CEfanmin** [comment]
I think after function load_data(), maybe should add:
X[np.isnan(X)] = 0.
then it works!

**Comment by danielegrattarola** [comment]
This is fixed now, thanks for opening the issue.
